src/sql/workbench/contrib/notebook/browser/cellViews/markdownToolbar.component.ts

```diff
@@ -28,10 +28,10 @@ export class MarkdownToolbarComponent {
 	public buttonList = localize('buttonList', "List");
 	public buttonOrderedList = localize('buttonOrderedList', "Ordered list");
 	public buttonImage = localize('buttonImage', "Image");
-	public buttonPreview = localize('buttonPreview', "Markdown preview toggle - off");
 
 	@Input() public cellModel: ICellModel;
 	private _actionBar: Taskbar;
+	private _togglePreviewAction: TogglePreviewAction;
```

Comment:
This doesn't seem to be necessary to have as a member variable - the only thing that uses it is the constructor so you can just create it there directly.

Reply:
Removed